static/templates/actions_popover_content.handlebars

```diff
@@ -21,6 +21,15 @@
     </a>
   </li>
 
+  {{#if message.edit_history}}
+  <li>
+    <a href="#" class="view_edit_history" data-msgid="{{message.id}}">
+      <i class="fa-history"></i>
+        {{t "View message history" }}
```

Comment:
I notice we still are missing an icon here; updating font-awesome makes sense to me!